tools/BUILD.bazel

```diff
@@ -16,6 +16,10 @@ platform(
           name: "dockerAddCapabilities"
           value: "SYS_ADMIN"
       }
+      properties: {
+          name: "dockerNetwork"
+          value: "standard"
```

Comment:
I just added this to demonstrate that we _could_ enable internet access in the remote executors. It's usually not recommended though as relying on external resources doesn't play well with the hermeticity concept.

There are multiple approaches for accessing a page here:

* Using an inline `data:` URL to render out a test page
* Setting up a dev server together with running the test. That might be more ideal but I'd need to look how we can achieve it.